model/search/ResultsWatcher.php

```diff
@@ -114,4 +129,11 @@ public function isResultSearchEnabled()
     {
         return (bool)$this->getOption(self::OPTION_RESULT_SEARCH_FIELD_VISIBILITY);
     }
+
+    private function transformDateTime(string $getStartTime): string
+    {
+        $timeArray = explode(" ", $getStartTime);
+        $date = DateTimeImmutable::createFromFormat('U', $timeArray[1]);
```

Comment:
Sure this will be always a timestamp? If not `$date` will be false and we will have an error.

Reply:
```
        if ($date === false) {
            $date = DateTimeImmutable::createFromFormat('Y-m-d H:i:s', $getStartTime);
        }
```
I think I should throw an error when  `$date` is still false. WDYT @gabrielfs7

Reply:
Yes, you can validate all `supported formats`, but "not" throw an exception. Instead `log` error, cause it should not break the interface for the user.

Reply:
https://github.com/oat-sa/extension-tao-outcomeui/pull/357/commits/b24717e95e8f08fd8ece834ce7f315199f2eb711